components/Recipe/Buttons/slideshow.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { MouseEventHandler } from 'react'
+
+interface Props {
+    name: string
+    handleClick: MouseEventHandler
+    shouldRender: boolean
+}
+
+const SlideShowButton: React.FC<Props> = ({ name, handleClick, shouldRender}) => {
+    let className: string = "w-48 my-4 py-2 px-4 rounded-full ";
+    if (shouldRender) className += "bg-blue text-black";
+    else className += "invisible";
+
+    return (
+        <button className={className} onClick={handleClick}>
+            {name}
+        </button>
+    )
+}
+
+export default SlideShowButton;
```

Comment:
We're going with no semicolons for formatting, so check that Prettier settings have them turned off